packages/compass-indexes/src/components/index-header-column/index-header-column.jsx

```diff
@@ -1,11 +1,15 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-
-import classnames from 'classnames';
-import styles from './index-header-column.module.less';
+import { css, cx, uiColors } from '@mongodb-js/compass-components';
 
 import { DEFAULT, DESC, ASC } from '../../modules/indexes';
 
+const sortStyles = css({
+  display: 'inline-block',
+  fontSize: '13px',
```

Comment:
Would it be possible to avoid a hardcoded number here? Maybe to use something from typography or value from `compass-font-sizes`.

Reply:
i ended up removing this as it didn't have an effect. plus, in the next PR as I'll use LG table for this, this whole will be gone.